example_sns_init.yaml

```diff
@@ -25,104 +31,307 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# The structure allows for mapping identities to human-readable names.
```

Comment:
I'd leave out the comment on this line given that the field must be left empty for now.